charts/nexus-iq/templates/deployment.yaml

```diff
@@ -28,6 +28,23 @@ spec:
       serviceAccountName: {{ include "iqserver.serviceAccountName" . }}
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      {{- if .Values.fixOwner.enabled }}
+      initContainers:
+        - name: chown-vols
+          image: "busybox:latest"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          securityContext:
+            runAsUser: 0
+            allowPrivilegeEscalation: true
+          command: ['sh', '-c', 'chown -R {{ .Values.fixOwner.updateUser }}:{{ .Values.fixOwner.updateGroup }} /var/log/nexus-iq-server {{ .Values.configYaml.sonatypeWork }}']
```

Comment:
when i was writing the unit test, i found `updateUser` was unset, since it's called `updateOwner` in values.yaml. Please match them up with whatever name you prefer.